sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -411,6 +411,42 @@ object RapidsConf {
       .integerConf
       .createWithDefault(2)
 
+  val GPU_COREDUMP_DIR = conf("spark.rapids.gpu.coreDump.dir")
+    .doc("The URI to a directory where a GPU core dump will be created if the GPU encounters " +
```

Comment:
we may want to explicitly say both local and distributed filesystems supported just so its super clear to users